rviz_assimp_vendor/rviz_assimp_vendor-extras.cmake.in

```diff
@@ -8,6 +8,11 @@ endif()
 
 find_package(assimp REQUIRED QUIET)
 
+# workaround bug in Assimp 4.1.0 https://github.com/assimp/assimp/issues/1914
+# Affecting Ubuntu package: libassimp-dev 4.1.0~dfsg-3, Brew: assimp 4.1.0
+string(REPLACE "/lib/lib/" "/lib/" ASSIMP_LIBRARY_DIRS "${ASSIMP_LIBRARY_DIRS}")
+string(REPLACE "/lib/include" "/include" ASSIMP_INCLUDE_DIRS "${ASSIMP_INCLUDE_DIRS}")
```

Comment:
The pattern should have a trailing slash to avoid matching in undesired positions.

Reply:
Yeah I had the same feeling, unfortunately it cannot have the trailing slash as the path exported by the assimp debian doesnt have one "/usr/lib/include"
```
CMake Error in CMakeLists.txt:
  Imported target "rviz_rendering::rviz_rendering" includes non-existent path

    "/usr/lib/include"
```
Example of job failing with the trailing slash in the pattern : https://ci.ros2.org/job/ci_linux/4579/

Reply:
The pattern should test for the end-of-string then?

Reply:
Looking at their code, it appears that end of string should work in all cases :+1:

https://github.com/ros2/rviz/pull/288/commits/a77a0404bd69c3bee4e0284ccee19f552c15ac9f

Reply:
:+1: